SOURCE/Main_DOODZ.c

```diff
@@ -763,20 +764,26 @@ int main( int nargs, char *args[] ) {
             // If Picard 2 Newton is activated: force initial Newton step
             if ( Newton == 1 && Nmodel.Picard2Newton == 1 ) {
                 model.Newton = 0;
-                first_Newton = 1;
+                FirstNewton  = 1;
             }
             
             while ( Nmodel.nit <= Nmax_picard && nstag<model.nstagmax) {
                 
                 if ( Nmodel.nit > 0 && Nmodel.Picard2Newton == 1 ) {
                     if ( rx_rel[Nmodel.nit-1] < Nmodel.Pic2NewtCond || rz_rel[Nmodel.nit-1] < Nmodel.Pic2NewtCond || rp_rel[Nmodel.nit-1] < Nmodel.Pic2NewtCond || Nmodel.nit>= Nmodel.nit_Pic_max) {
-                        if ( first_Newton == 1 ) cholmod_free_factor ( &CholmodSolver.Lfact, &CholmodSolver.c);
-                        if ( first_Newton == 1 ) CholmodSolver.Analyze = 1;
-                        if ( first_Newton == 0 ) CholmodSolver.Analyze = 0;
-                        if ( first_Newton == 1 ) first_Newton = 0;
+                        if ( FirstNewton == 1 ) cholmod_free_factor ( &CholmodSolver.Lfact, &CholmodSolver.c);
```

Comment:
Is `cholmod_free_factor` a function? Functions names should represent an action that will be performed. 
Also as we agreed let's keep to the Google CPP naming conventions: https://google.github.io/styleguide/cppguide.html#Functions

Reply:
Let's discuss the options live on Monday